heronpy/streamlet/BUILD

```diff
@@ -3,7 +3,7 @@ package(default_visibility = ["//visibility:public"])
 load("/tools/rules/pex/pex_rules", "pex_library", "pex_binary")
 
 pex_library(
-    name = "heron-pythondsl-py",
+    name = "heron-python-streamlet-api-py",
```

Comment:
why streamletapi? why not just streamlet?

Reply:
Works for me either way. I'll shorten it to just streamlet.